meteor/server/migration/X_X_X.ts

```diff
@@ -607,4 +657,108 @@ export const addSteps = addMigrationSteps(CURRENT_SYSTEM_VERSION, [
 			)
 		},
 	},
+
+	{
+		id: `Studio playoutDevices ensure new DeviceType values`,
+		canBeRunAutomatically: true,
```

Comment:
this should be marked as dependantOn the other one which converts the old device config layout to it being on the studio.